SignNow.Net/_Internal/Helpers/UserAgentSdkHeaders.cs

```diff
@@ -0,0 +1,191 @@
+using System;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("SignNow.Net.Test")]
+namespace SignNow.Net.Internal.Helpers
+{
+
+#if NET45
+    using System.Reflection;
+    using Microsoft.Win32;
+#else
+    using System.Runtime.InteropServices;
+    using System.Text.RegularExpressions;
+#endif
+
+    internal static class UserAgentSdkHeaders
+    {
+        /// <summary>
+        /// Returns SDK client's name
+        /// </summary>
+        /// <returns></returns>
+        public static string ClientName()
+        {
+            return "SignNow .NET API Client";
+        }
+
+        /// <summary>
+        /// Returns SDK version (e.g: v1.0.0.0)
+        /// </summary>
+        /// <returns></returns>
+        public static string SdkVersion()
+        {
+            var assemblyFQN = typeof(UserAgentSdkHeaders).AssemblyQualifiedName.Split(',');
+            var version = assemblyFQN.GetValue(2).ToString().Trim();
```

Comment:
You should search for Version= instead of index.  Something like IndexOf("Version=")...